### ISSUE TITLE
Upgrade vuetify from 1.* to 2.* ?

### ISSUE THREAD

**Issue by hfoffani** [fastapi/full-stack-fastapi-template#426]
I would like to upgrade the vuetify components from 1.* to 2.*
Has anyone done it?
I am getting conflicts between npm packages (webpack, sass-loader, etc.)

**Comment by AEAMcNett** [comment]
This is documented fairly thoroughly on the Vuetifyjs site: [https://vuetifyjs.com/en/getting-started/upgrade-guide/](https://vuetifyjs.com/en/getting-started/upgrade-guide/)

This linter is helpful: [https://github.com/vuetifyjs/eslint-plugin-vuetify](https://github.com/vuetifyjs/eslint-plugin-vuetify)

**Comment by hfoffani** [comment]
Yes, I've followed the guide, thanks. However I couldn't find the packages conflict yet. Probably due to lack of enough knowledge here.
I asked in the chance that someone had already made the port of this project and published the diff or fork.
Thanks anyway!

**Comment by YuriiMotov** [comment]
Outdated
